<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IEEE80211g</class>
 <widget class="QMainWindow" name="IEEE80211g">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IEEE80211g</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1011</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxPosX">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxPosY">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton">
           <property name="text">
            <string>Launch Ray-Tracing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="antena">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/path4136.png</normaloff>:/path4136.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="receiver">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="settingsButton">
           <property name="text">
            <string>settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="helpButton">
           <property name="text">
            <string>help</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="sceneRect">
        <rectf>
         <x>0.000000000000000</x>
         <y>0.000000000000000</y>
         <width>969.000000000000000</width>
         <height>505.000000000000000</height>
        </rectf>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="generateCoveragePushButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>540</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Coverage</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBoxResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>1009</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>1009</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance Emettor - Receptor (m)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>420</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Prx">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>430</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="DelaySpread">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>470</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="RiceFactor">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>490</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QToolButton" name="plotButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>590</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="Distance">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>450</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>150</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Penetration depth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="commerceUp">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>170</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rue du Commerce Up:</string>
    </property>
   </widget>
   <widget class="QLabel" name="deuxEg">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>190</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rue de deux Eglises:</string>
    </property>
   </widget>
   <widget class="QLabel" name="spa">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>210</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rue de Spa:</string>
    </property>
   </widget>
   <widget class="QLabel" name="commerceDown">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>230</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rue du Commerce Down: </string>
    </property>
   </widget>
   <widget class="QLabel" name="indu">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>250</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rue de l'Indusctrie: </string>
    </property>
   </widget>
   <widget class="QLabel" name="science">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>270</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rue de la Science: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1051</x>
      <y>1</y>
      <width>174</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Transmitted power (dBm)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="powerResultSpinBox">
    <property name="geometry">
     <rect>
      <x>1051</x>
      <y>24</y>
      <width>133</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1051</x>
      <y>57</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum Binary debit (Mb/s)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="binaryResultsSpinBox">
    <property name="geometry">
     <rect>
      <x>1051</x>
      <y>80</y>
      <width>179</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="clearWorkspaceButton">
    <property name="geometry">
     <rect>
      <x>1051</x>
      <y>113</y>
      <width>132</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Workspace</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1051</x>
      <y>169</y>
      <width>219</width>
      <height>231</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>textBrowser_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>generateCoveragePushButton</zorder>
   <zorder>spinBoxResult</zorder>
   <zorder>label_5</zorder>
   <zorder>Prx</zorder>
   <zorder>DelaySpread</zorder>
   <zorder>RiceFactor</zorder>
   <zorder>plotButton</zorder>
   <zorder>Distance</zorder>
   <zorder>textBrowser</zorder>
   <zorder>label_6</zorder>
   <zorder>commerceUp</zorder>
   <zorder>deuxEg</zorder>
   <zorder>spa</zorder>
   <zorder>commerceDown</zorder>
   <zorder>indu</zorder>
   <zorder>science</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
